src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/allelespecific/ReducibleAnnotation.java

```diff
@@ -26,6 +30,28 @@
                                                         final VariantContext vc,
                                                         final ReadLikelihoods<Allele> likelihoods);
 
-    @SuppressWarnings({"unchecked", "rawtypes"})//FIXME: there's no way to properly use generics here because of method overriding
-    public abstract void calculateRawData(VariantContext vc, final ReadLikelihoods<Allele> likelihoods, ReducibleAnnotationData rawAnnotations);
```

Comment:
In GATK3 as it turned out we were never using calculateRawData independently anywhere. Ever invocation of the method was somewhere in the process of calling annotateRawData so it could be removed from the interface.